### PR TITLE
Add hash column to the draft and published databases

### PR DESCRIPTION
**Pull request by missamynicholson**
Adding a database migration to append a hash column to both draft and publish tables.

### CODE REVIEW

**Comment by peteclark-ft** [comment]
Do we also need to change the `config.yml` to write the new hash header for inbound PUT requests?

**Comment by missamynicholson** [comment]
@peteclark-ft Yes indeed! I branched from master which didn't have that file until today :joy:

**Comment by izzyblues** [comment]
@missamynicholson @peteclark-ft Changing the `config.yml` is not necessary for upgrading the schema version in Aurora.

File: db/schema.go [hunk 1]
```diff
@@ -47,6 +47,16 @@ var (
 		drop table draft_annotations;
 		`,
 		},
+		{2, "add-hash-annotations-tables",
```
Comment (izzyblues) [inline db/schema.go]: I forgot to tell you that golang has a tool to automatically format properly your code.
To fix the code format you need to run `go fmt` before committing your code.

Comment (izzyblues) [reply db/schema.go]: Sorry. The formatting is correct. 😅

**Pull request merged**